packages/cli/src/AbstractServer.ts

```diff
@@ -167,7 +167,7 @@ export abstract class AbstractServer {
 
 		await this.setupHealthCheck();
 
-		console.log(`n8n ready on ${ADDRESS}, port ${PORT}`);
+		this.logger.info(`n8n ready on ${ADDRESS}, port ${PORT}`);
```

Comment:
Logging changes like these ensure `N8N_LOG_LEVEL=silent` mutes this logging noise during benchmarking runs.